recipes/opendrift/conda_build_config.yaml

```diff
@@ -0,0 +1,2 @@
+channel_targets:
+  - conda-forge opendrift_rc
```

Comment:
@gauteh when you release the stable version just remove this file from the feedstock and re-render. That is enough to make the stable version go to main instead of opendrift_rc. You can keep an RC branch if you want for future RC releases.

Reply:
Ok, thank you very much.